terminology.asciidoc

```diff
@@ -210,4 +210,4 @@ Parties also will need to oblige to local legislation.
 
 When a new version of OCPI is implemented, the data exchanged via the old version does not have to be available via the newer version of OCPI.
 Hence, the Version end-point will probably have different end-points per version. So when an object is stored with a URL that contains a version,
-it is NOT REQUIRED to be available at a URL with a different version number.
```

Comment:
What changed here? Seems like the same line, watch out with adding/removing white spaces or other non printed characters

Reply:
This is just a new line at annd of line added.